### PR TITLE
i18n: fix comments from PR #425

### PR DESCRIPTION
**Pull request by carygravel**

### CODE REVIEW

**Comment by carygravel** [comment]
I hope this fixes the comments from PR #425. Note that the DE docs should also probably switch to relative links.

File: i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx [hunk 1]
```diff
@@ -467,8 +467,8 @@ vehicles:
     type: template
     template: tesla
     title: # Will be displayed in the user interface (optional)
-    accessToken: # See https://docs.evcc.io/docs/devices/vehicles#tesla
-    refreshToken: # See https://docs.evcc.io/docs/devices/vehicles#tesla
+    accessToken: # See https://docs.evcc.io/en/docs/devices/vehicles#tesla
+    refreshToken: # See https://docs.evcc.io/en/docs/devices/vehicles#tesla
```
Comment (naltatis) [inline i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx:471]: This is autogenerated. The source is in evcc core. Can you make small PR there?

**Review by naltatis — APPROVED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx (hunk 1 above)
Comment (carygravel) [reply i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx:471]: Are you suggesting that it should be a relative link? If so, does that make sense in a comment?

If not, what change are you looking for here?

Comment (naltatis) [reply i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx:471]: No, we can't use relative links here. The same text is used in `evcc configure` and will be used in the Web Config UI.

The only thing I wanted to say is, that the code you've edited is autogenerated and will be overwritten. The source is here: https://github.com/evcc-io/evcc/blob/master/templates/definition/vehicle/tesla.yaml#L29

Comment (naltatis) [reply i18n/en/docusaurus-plugin-content-docs/current/devices/vehicles.mdx:471]: I've made this change here. There were a bunch of other locations that needed adjustment as well.
https://github.com/evcc-io/evcc/pull/10053

**Pull request merged**